### DISCUSSION TITLE
WebP security fixed?

### DISCUSSION THREAD

**Discussion by gkhar2** [sylikc/jpegview#323] [General]
Hello, 

in 2023 there was a security issue with the webp lib. 

Is jpegview affected by this and is it already fixed?

Here is a link to heise.de where this issue is explained.

Thank you.

https://www.heise.de/news/Unzaehlige-Anwendungen-betroffen-Chaos-bei-WebP-Luecke-9319783.html

**Comment by qbnu** [comment]
Yes, it's fixed in the latest release (1.3.46)

**Reply by gkhar2** [reply]
great, thanks
